utility/src/test/java/org/oppia/android/util/networking/NetworkConnectionUtilDebugModuleTest.kt

```diff
@@ -0,0 +1,101 @@
+package org.oppia.android.util.networking
+
+import android.app.Application
+import android.content.Context
+import androidx.test.core.app.ApplicationProvider
+import androidx.test.ext.junit.runners.AndroidJUnit4
+import com.google.common.truth.Truth.assertThat
+import dagger.BindsInstance
+import dagger.Component
+import dagger.Module
+import dagger.Provides
+import org.junit.Before
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.oppia.android.util.logging.EnableConsoleLog
+import org.oppia.android.util.logging.EnableFileLog
+import org.oppia.android.util.logging.GlobalLogLevel
+import org.oppia.android.util.logging.LogLevel
+import org.robolectric.annotation.Config
+import org.robolectric.annotation.LooperMode
+import javax.inject.Inject
+import javax.inject.Singleton
+
+/** Tests for [NetworkConnectionUtilDebugModule]. */
+@RunWith(AndroidJUnit4::class)
+@LooperMode(LooperMode.Mode.PAUSED)
+@Config(manifest = Config.NONE)
+class NetworkConnectionUtilDebugModuleTest {
+
+  @Inject
+  lateinit var networkConnectionUtil: NetworkConnectionUtil
+
+  @Before
+  fun setUp() {
+    setUpTestApplicationComponent()
+  }
+
+  @Test
+  fun testNetworkConnectionUtilProdModule_prodImplementationIsInjected() {
+    assertThat(networkConnectionUtil is NetworkConnectionUtilDebugImpl).isTrue()
+  }
+
+  private fun setUpTestApplicationComponent() {
+    DaggerNetworkConnectionUtilDebugModuleTest_TestApplicationComponent.builder()
+      .setApplication(ApplicationProvider.getApplicationContext())
+      .build()
+      .inject(this)
+  }
+
+  // TODO(#89): Move this to a common test application component.
+  /** Test specific dagger module for [NetworkConnectionUtilDebugModuleTest]. */
```

Comment:
For here & other test files: I suggest removing the extra KDocs (the top-level suite KDoc is fine to keep) & adding exemptions. We should actually be exempting all test files for KDocs by default.

Reply:
Extra Kdocs removed and test files added to exemption.